web/themes/joinup/prototype/sass/layouts/sections/_section--footer.sass

```diff
@@ -5,4 +5,16 @@
 // Component
 // --------------------------------------------------
 .section--footer
-  +hidden-print
+  text-align: right
+  ul
```

Comment:
Thanks for adding the hidden print back.
I'm leaving the rest of the comments about the footer aside as for a related pull request for more refactoring in this component is in the queue.